### DISCUSSION TITLE
Oculus Go with Nolo CV1 Wireless/Wired?

### DISCUSSION THREAD

**Discussion by Grumpyn** [alvr-org/ALVR#882] [Q&A]
Hello, support this software (ALVR) Oculus Go with Nolo CV1 tracking? Wired or Wireless? I'm looking for an alternative to Nolo Home its a garbage...

Thx.

**Comment by zmerp** [comment]
I'm not sure ALVR will support Nolo tracking. But I know that Rifcat Vridge supports it. It may have greater latency than ALVR though.

**Reply by Grumpyn** [reply]
NoloVR is not officialy suppored by Riftcat, when i try edit config.cfg to enable Nolo it does not work. I tried ALVR and working great even in poor network,  but without Nolo tracking support it useless for Go. 

(sorry for my english)

**Reply by zmerp** [reply]
Unfortunately the next version of ALVR will use Vulkan 1.1 and and Oculus Go support will be phased out. We may re-add support in the future but it's not at the top of our priority list. Even if we keep support it's unlikely we can spend time on supporting accessories for old headsets

**Reply by Grumpyn** [reply]
OK. In the meantime, I'll survive with Nolo Home and eventually move on to Quest.  Thx for your support :-)
